Code/EnginePlugins/PhysXPlugin/WorldModule/Implementation/PhysXWorldModule.cpp

```diff
@@ -581,7 +581,9 @@ void ezPhysXWorldModule::Initialize()
 
     desc.flags |= PxSceneFlag::eENABLE_ACTIVE_ACTORS;
     desc.flags |= PxSceneFlag::eEXCLUDE_KINEMATICS_FROM_ACTIVE_ACTORS;
-    desc.flags |= PxSceneFlag::eENABLE_KINEMATIC_PAIRS;
+    desc.kineKineFilteringMode = PxPairFilteringMode::eKEEP;
+    //TODO: Do we need this? PhysX4 upgrade.
+    //desc.staticKineFilteringMode = PxPairFilteringMode::eKEEP;
```

Comment:
I can't think of a usecase for static <-> kinematic pairs, so we don't need it atm.

Reply:
Ok removed.